build.sbt

```diff
@@ -13,6 +13,9 @@ ThisBuild / developers ++= List(
 ThisBuild / githubWorkflowJavaVersions ++=
   Seq(JavaSpec.temurin("17"), JavaSpec(JavaSpec.Distribution.GraalVM("latest"), "17"))
 
+ThisBuild / githubWorkflowBuildTimeoutMinutes := Some(60)
```

Comment:
These two changes are not necessary. They are here mainly for showcase purposes.